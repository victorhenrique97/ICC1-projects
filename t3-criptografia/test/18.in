7.bin
2
