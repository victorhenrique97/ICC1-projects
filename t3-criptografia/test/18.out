Vamos ver se voce consegue contar as ocorrencias disso: gaggagagag gagagag. Nao adianta botar a culpa no 9gag se voce nao conseguir terminar o trabalho a tempo. Ao menos usem reddit ao inves do 9gag.
